README.md

```diff
@@ -573,6 +573,22 @@ The config map includes the following:
 
 	`internal` is the default type, anything else is considered external. `auth-type` could include strings like: ipa, ldap, active_directory, saml or simply custom.
 
+* The httpd configuration type `auth-configuration`, default is `internal`
+
+	This parameter drive which configuration files httpd will load upon start-up. Supported values are:
+
+
+	| Value    | External-Authentication Configuration |
+	| ---------|---------------------------------------|
+	|internal | Application Based Authentication (_default_) - Database, Ldap/Ldaps, Amazon |
+	| external | IPA, IPA 2-factor authentication, IPA/AD Trust, Ldap (OpenLdap, RHDS, Active Directory, etc.)
+	| active-directory | Active Directory domain realm join
+	| saml | SAML based authentication (Keycloak, etc.)
+
+* The kerberos realms joined `auth-kerberos-realms`, default is `undefined`
```

Comment:
"The kerberos realms joined" seems strangely worded to me. Maybe "The kerberos realm to join"?

Also if this is indeed plural, how do you specify multiple? Comma separated?